docs/changelog/CHANGELOG.md

```diff
@@ -20,6 +20,8 @@ See also [LTS releases](https://docs.victoriametrics.com/lts-releases/).
 
 * FEATURE: [Single-node VictoriaMetrics](https://docs.victoriametrics.com/) and [vmstorage](https://docs.victoriametrics.com/cluster-victoriametrics/): improve startup times when opening a storage with the [retention](https://docs.victoriametrics.com/#retention) exceeding a few months.
 
+* BUGFIX: [vmui](https://docs.victoriametrics.com/#vmui): replaced `await navigator.clipboard.writeText` with `useCopyToClipboard` hook. See [this pull request](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/7778).
```

Comment:
This changelog isn't user-oriented. If the change is minor, let's  omit it in changes. Or if it is important, try explaining it from the user perspective

Reply:
@hagen1778 I updated the changelog. This change adds clear error messages for clipboard failures, helping users troubleshoot.